doc/guide/install.rst

```diff
@@ -246,6 +246,14 @@ On macOS and on Linux it is pre-installed. On Windows, it can easily be installe
 
 Instructions for running the test scripts can be found in the :doc:`Testing Guide <Guide_Testing>`.
 
+The developer dependencies also include `pre-commit <https://pre-commit.com/#intro>`_, which is used to run pre-commit hooks, such as code linting and formatting, when creating a new commit. In order to use the hooks defined in ``.pre-commit-config.yaml``, you need to install the hooks by running (you can also install [this extension](https://marketplace.visualstudio.com/items?itemName=elagil.pre-commit-helper) if you use Visual Studio Code):
+
+.. code-block:: shell
+
+   pre-commit install
+
+We strongly recommend you read :ref:`What happens with pre-commit hooks? <pre-commit_hooks>` if you do so.
+
```

Comment:
A question here: shouldn't it be also mentionned to install the black formatter? As the pre-commit are used to apply changes suggested by the formatter..

Reply:
If it is required - is it or does pre-commit take care of that? - I suppose that it shouldn't be mentioned but listed in `extra_require["dev"]` of `setup.py`.

Reply:
Pre-commit takes care of it, see my https://github.com/CLIMADA-project/climada_python/pull/892#discussion_r1683039104